src/wp-content/themes/twentyeleven/functions.php

```diff
@@ -741,6 +745,8 @@ function twentyeleven_footer_sidebar_class() {
 	 *
 	 * @since Twenty Eleven 1.0
 	 *
+	 * @global WP_Comment $comment Global comment object.
```

Comment:
[58715.2.patch](https://core.trac.wordpress.org/attachment/ticket/58715/58715.2.patch) documented the `$comment` global in `twentyten_comment()`, `twentyeleven_comment()` and `twentytwelve_comment()`. This seems odd when `$comment` is also a parameter, though I do not know of a reason why these functions cannot document it as both global and parameter.